features/steps/then.php

```diff
@@ -145,6 +145,16 @@ function ( $world, $stream ) {
 	}
 );
 
+$steps->Then( '/^(STDOUT|STDERR) should be a version string (<|<=|>|>=|==|=|!=|<>) ([+\w\.-]+)$/',
```

Comment:
This will eventually want to end up in WP-CLI's test suite, if we deem it a necessary test to keep.